doc/api/http.md

```diff
@@ -2141,9 +2141,13 @@ the request body should be sent. See the [`'checkContinue'`][] event on
 
 <!-- YAML
 added: REPLACEME
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/44820
+    description: Allow passing hints as an object.
 -->
 
-* `links` {string|Array}
+* `links` {string|Array|Object}
```

Comment:
Just to express my comment once again.
I think `writeEarlyHints` should only accept `object` and should treat `Link` as special case.

The previous PR already have concise on it https://github.com/nodejs/node/pull/44180#issuecomment-1219808821

Reply:
For now, it's better to accept all of them. After it, we could create a new PR which will be tagged as a major (cause it will be a breaking change) keeping just the object support.

Reply:
I agree, let's do this.

Reply:
![Hello there](https://media.tenor.com/WuOwfnsLcfYAAAAS/star-wars-obi-wan-kenobi.gif)

I updated the pull-request.